betterproto/tests/inputs/import_service_input_message/test_import_service.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from betterproto.tests.mocks import MockChannel
+from betterproto.tests.output_betterproto.import_service_input_message.import_service_input_message import (
+    RequestResponse,
+    TestStub,
+)
+
+
+@pytest.mark.xfail(reason="Request Input Messages are not imported for service")
```

Comment:
Could be useful for the reason to reference the issue number.

Reply:
Good idea! Will do